src/ext/issuance.js

```diff
@@ -38,8 +38,8 @@ const COMMITMENT_URL = "https://raw.githubusercontent.com/privacypass/ec-commitm
  */
 function signReqCF(url, details) {
     const reqUrl = url.href;
-    const captchaResp = reqUrl.includes(requestIdentifiers()["query-param"]);
-    const alreadyProcessed = reqUrl.includes(requestIdentifiers()["post-processed"]);
+    const captchaResp = url.searchParams.has(requestIdentifiers()["query-param"]);
```

Comment:
should be get instead of has

Reply:
I think `has` is fine because we don't modify this part of the URL and it is used to send the subsequent XHR request, no?